config/settings.yml

```diff
@@ -133,6 +133,7 @@
     :refresh_interval: 15.minutes
   :openshift:
     :refresh_interval: 15.minutes
+    :get_container_images: true
```

Comment:
note that if you backport / make a hotfix, i believe euwe had both openshift & openshift-enterprise types, both need this true default.